README.md

```diff
@@ -185,6 +185,7 @@ then Lodash/Underscore is the better option.*
 
 1. [_.inRange](#_inRange)
 2. [_.random](#_random)
+3. [_.clamp](#clamp)
```

Comment:
probably better to follow alphabetic order for the first charactor after . like how we order the names in other sections.